requirements.txt

```diff
@@ -15,6 +15,7 @@ mistune>=0.8.4  # package
 numpy>=1.14.1  # package
 pandas>=0.23.0  # package
 pyparsing>=2.4,<3  # package
+pypd==1.1.0  # package
```

Comment:
Let's not make pypd a requirement for all the users who are not PagerDuty customers. Please move this line from `requirements.txt` to `requirements-dev.txt` that are required for the tests to pass. Also, see my edit suggestions in the action implementation that make sure that the pypd module is optional.